content/es/working-groups.md

```diff
@@ -0,0 +1,279 @@
+
+No hay un conjunto específico de requerimientos o calificaciones la membresía del GT más allá de estas reglas.
+
+El GT puede añadir miembros adicionales al GT por consenso unánime.
+
+Un miembro del GT puede ser eliminado del GT por resignación voluntaria, o por consenso unánime de toda la lista de miembros del GT.
+
+Cambios en la membresía del GT deben de ser puestos en la agenda, y pueden ser sugeridos como cualquier otro item de esta (véase "Reuniones del GT" más abajo).
+
+Si una adición o supresión es propuesta durante una reunión, y el GT no ha asistido al completo, entonces la adición o supresión se añade a la agenda de la reunión siguiente. Esto es debido a que todos a los miembros se le da la oportunidad de participar en todas las decisiones de membresía. Si un miembro del GT no está disponible para atender a la reunión cuando es planeada una decisión de membresía, entonces su consentimiento es asumido.
+
+No más de 1/3 de los miembros del GT han de estar afiliados a la mismo empleador.  Si una adición o supresión o resignación de un miembro del GT, o un cambio de empleo de un miembro del GT, crea una situación donde más 1/3 de la membresía del GT comparte un empleador, entonces la situación debe ser remediada inmediatamente con la resignación o supresión de uno o más miembros del GT afiliados con el empleador(es) más representado(s).
+
+### Reuniones del GT
+
+Los GT se reúnen semanalmente en un Google Hangout On Air. Un moderador designado aprobado por el GT lleva a cabo la reunión. Cada reunión debe ser publicada en YouTube.
+
+Se añaden items a la agenda del GT que sean considerados contenciosos o sean modificaciones en el gobierno, política de contribución, membresía del GT, o proceso de publicación.
+
+La intención de la agenda no es la de aprobar o revisar todos los parches; lo que ha de ocurrir de forma continua en GitHub y ha de ser llevada a cabo por un grupo más grande de Colaboradores.
+
+Cualquier miembro de la comunidad o colaborador puede preguntar si algo puede ser añadido a la agenda de la siguiente reunión abriendo un GitHub Issue. Cualquier colaborador, miembro de GT o moderador puede añadir un item a la agenda añadiendo la etiqueta **Agenda del GT** al issue.
+
+Con anterioridad a cada reunión del GT el moderador compartirá la Agenda con los miembros del GT. Los miembros del GT pueden añadir tantos items como deseen al principio de cada reunión. El moderador y el GT no pueden vetar o suprimir items.
+
+El GT puede invitar personas o representantes a ciertos proyectos para participar en capacidad de no votante.
+
+El moderador es responsable de resumir la discusión de cada item en la agenda y enviar una pull request después de la reunión.
+
+### Proceso de Búsqueda de Consenso
+
+El GT sigue un proceso de [Búsqueda de Consenso](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
```

Comment:
What about this Wikipedia article? (in spanish) http://es.wikipedia.org/wiki/Decisi%C3%B3n_por_consenso